docs/UserGuide.md

```diff
@@ -38,11 +43,51 @@ It is optimized for fast typists and replicates a Command Line Interface (CLI) s
 
    * **`exit`** : Exits the app.
 
-6. Refer to the [things to note](#3-things-to-note) below for details of each command.
+6. You can refer to the [things to note](#3-things-to-note) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+## 3. About Our User Guide
 
-## 3. Things to note
+This guide serves to:<br>
+
+1. Educate new users on how to get started with WorkBook!
+2. Explain the purpose and how to use each and every command!
+3. Display a summary of all the commands you can utilize!
```

Comment:
can probably save the exclamation marks for the top part when welcoming them, here seems abit excessive